bindings/python/quantity.py

```diff
@@ -0,0 +1,175 @@
+from typing import Any
+import numpy as np
+HAS_PINT = True
+try:
+    import pint
+except Exception:
+    HAS_PINT = False
+
+DLITE_QUANTITY_TYPES = [
+    'int', 'float', 'double', 'uint',
+    'int8', 'int16', 'int32', 'int64',
+    'uint8', 'uint16', 'uint32', 'uint64',
+    'float32', 'float64', 'float80', 'float96', 'float128'
+]
+quantity_helper = None
+
+
+class QuantityHelper:
+
+    def __init__(self):
+        self.__dict__['_instance'] = None
+        self.__dict__['_registry'] = None
```

Comment:
Why can't you just write it like 

    self._instance = None
    self._registry = None

Reply:
because I'm overwriting the method __setattr__, and doing ```self._instance = None``` will raise a RecursionError